src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,622 @@
+      },
+      "ketamine": {
+        "PharmaceuticalName": "ketamine",
+        "PharmaceuticalDoseAmount": 0.1,
+        "PharmaceuticalDoseUnit": "ug/kg/min"
+      }
+    },
+    "Sample": {
+      "BodyPart": "BRAIN",
+      "BodyPartDetails": "Motor Cortex"
+    },
+    "ProbeContours": {
+      "p021": {
+        "Contour": [
+          [0, 0, 0],
+          [0, 10, 0],
+          [1, 11, 0],
+          [2, 10, 0],
+          [2, 0, 0]
+        ],
+        "Unit": "mm"
+      }
+    },
+    "TaskName": "Reach-to-Grasp",
+    "TaskDescription": "A task that involves the reaching of an object and holding it for a specific time"
+  }
+}
+```
+
+## Recording Events (*_events.tsv)
+
+The `*_events.tsv` and corresponding `*_events.json` sidecar files are OPTIONAL and can be used to
+indicate time points of recording events. Each task events file requires a corresponding task data
+file. These events can be internal recording system events, task-related events, or events triggered
+by the experimentalist (for example, manual reward). Note that these events must share a common clock
+with the corresponding ephys recording data. For more details, see the
+Task Events documentation.
+Note that this file can also be used to describe stimulation performed during the recording. For this,
+please follow the iEEG stimulation documentation.
+
+
+## Multi-part recordings
+
+Two different procedures are supported to handle multi-part recordings. In short, the two options are:
+
+1. each recording is stored in an independent data file, and the corresponding metadata is described
+in the `*_scans.tsv` file; or
+2. several recordings are stored in a single data file, and the corresponding metadata is described
+in the `*_events.tsv` file.
+
+These two options are made available to support different usages and habits of the experimenters, as
+well as to benefit from the capability of the supported data formats (NWB and NIX).
+They are described in the following subsections, and made explicit through some of the example data sets.
+
+
+### Multiple tasks / runs in separate files (*_scans.tsv)
+
+The `*_scans.tsv` should be used to provide information about multiple parts of an acquisition
+session (for example, recording start times in case the recording was paused and restarted)
+when the data from each of these different recordings is stored in separate files.
+Each data file should have a name that contains a `_task-XX` and/or `_run-XX` suffix, and
+should be described by at most one row in the `*_scans.tsv` file. See also the BIDS Scans
+specification.
+Relative paths to files should be used under a compulsory “filename” header.
+If acquisition time is included, it should be with the “acq_time” header. Datetime should
+be expressed in the following format 2009-06-15T13:45:30 (year, month, day, hour (24h),
+minute, second; this is equivalent to the RFC3339 “date-time” format, time zone is always
+assumed as local time).
```

Comment:
I recalled we had some discussion but -- shouldn't we allow here timezone specification and assume "local" (which would be what? any placement to state?) only if not specified?

Reply:
Does this even play a role? Important is that within the same BIDS dataset time is specified within the same time zone, no?

Reply:
we should probably raise awareness that this information might be GDPR sensitive?

Reply:
- we might want to just refer to `Units` section in https://github.com/bids-standard/bids-specification/blob/fa58763066369de94c0753312417222119e37d3e/src/common-principles.md?plain=1#L987 describing time format.
- in the same file we have on shift of the dates for privacy protection: https://github.com/bids-standard/bids-specification/blob/fa58763066369de94c0753312417222119e37d3e/src/common-principles.md?plain=1#L1039 .  

I think we might want to make a separate PR that 
- would make that privacy protection of dates more prominent in that document
- make it a subsection we could refer to here (and possibly other places throughout BIDS)
- mention specifically GDPR and HIPPA

WDYT?